votes/templates/votes/stv_votescreen.html

```diff
@@ -65,7 +65,7 @@ <h2 class="card-header">Options</h2>
                     {% for choice in choices %}
                         <div class="card mb-3" data-voteid="{{ choice.formid }}">
                             <label class="mb-0 card-header" for="{{ choice.formid }}" class="">
```

Comment:
For whatever reason, this has ended up with two class blocks. Maybe nix one while you're at it